<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnPresentacion</class>
 <widget class="QWidget" name="vtnPresentacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alta Presentaci贸n</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>* Tipo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineUnidad_Medida">
         <property name="statusTip">
          <string>Ingrese unidad de medida (solo letras)</string>
         </property>
         <property name="accessibleDescription">
          <string>texto</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>* Unidad de Medida</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineTipo">
         <property name="statusTip">
          <string>Ingrese el tipo de Presentaci贸n (solo letras)</string>
         </property>
         <property name="accessibleDescription">
          <string>texto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBuscar">
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>* Cantidad de Fracciones</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QSpinBox" name="spinCantidad">
         <property name="statusTip">
          <string>Ingrese la cantidad del Producto para dicho lote</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupPresentacion">
     <property name="title">
      <string>Presentaci贸n</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableWidget" name="tablaPresentacion">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>160</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Tipo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Unidad de Medida</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Cantidad de Fracciones</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fraccionable</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Super Presentaci贸n</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupFraccionable">
     <property name="title">
      <string>Fraccionable</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tablaFraccionable">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>160</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Tipo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Unidad de Medida</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Cantidad de Fracciones</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineTipo</tabstop>
  <tabstop>btnBuscar</tabstop>
  <tabstop>lineUnidad_Medida</tabstop>
  <tabstop>spinCantidad</tabstop>
  <tabstop>tablaPresentacion</tabstop>
  <tabstop>tablaFraccionable</tabstop>
  <tabstop>btnAceptar</tabstop>
  <tabstop>btnCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
